dotcom-rendering/src/model/groupTrailsByDates.ts

```diff
@@ -194,5 +194,19 @@ export const groupTrailsByDates = (
 		}
 	}
 
+	// Sort - in cases where card order is unexpected (e.g not date ordered), it is possible this array will be out of order.
+	groupedTrails.sort((a, b) => {
```

Comment:
my memory of `sort` is always so fuzzy! Could we add a short comment to clarify whether we want to sort most recent items with the latest date first for instance?

Reply:
Added :)